specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2022-06-01-preview/examples/confidentialWatchlists/GetConfidentialWatchlists.json

```diff
@@ -0,0 +1,54 @@
+{
+  "parameters": {
+    "api-version": "2022-06-01-preview",
+    "subscriptionId": "d0cfe6b2-9ac0-4464-9919-dccaee2e48c0",
+    "resourceGroupName": "myRg",
+    "workspaceName": "myWorkspace",
+    "operationalInsightsResourceProvider": "Microsoft.OperationalInsights"
```

Comment:
>operationalInsightsResourceProvider [](http://example.com/codeflow?start=5&length=35)

I don't see this parameter is defined in the spec. If so, please remove it (from all examples)